syncd/FlexCounter.cpp

```diff
@@ -511,8 +512,82 @@ class CounterContext : public BaseCounterContext
         }
     }
 
+    void addVoqEntryToChassisAppDb(
+            _In_ const sai_object_id_t rid,
+            _In_ const sai_object_id_t vid,
+            _In_ std::vector<swss::FieldValueTuple> &counterValues,
+            _In_ swss::DBConnector &countersDb,
+            _In_ std::shared_ptr<swss::Table> cfgDeviceMetaDataTable,
+            _Out_ std::shared_ptr<swss::Table> appDbCountersTable ) {
+       // We need to update CHASSIS_APP_DB only if it is a VOQ counter. So first check
```

Comment:
please move { to new lina and add empty line every after } to make code readable

Reply:
Done